sdk/textanalytics/Azure.AI.TextAnalytics/tests/SessionRecords/TextAnalyticsClientLiveTests/RecognizeEntitiesCategoriesSubCategories.json

```diff
@@ -7,12 +7,12 @@
         "Content-Length": "233",
         "Content-Type": "application/json",
         "Ocp-Apim-Subscription-Key": "Sanitized",
-        "traceparent": "00-e7e2c5e43953244dba0af41bbd130d7f-4ed0fadaa6b5cc4e-00",
+        "traceparent": "00-4d4219468644864080f657dca87dffaf-fe6fc466ab3ae547-00",
```

Comment:
Do recorded tests need to be re-recorded?

Reply:
Yes. Only for the methods that changed. Because this included a renaming, the files needed to be generated again.